railties/lib/rails/generators/testing/assertions.rb

```diff
@@ -111,14 +111,14 @@ def assert_field_type(attribute_type, field_type)
           assert_equal(field_type, create_generated_attribute(attribute_type).field_type)
         end
 
-        # Asserts the given attribute type gets a proper default value:
+        # Asserts the given attribute type gets a proper example value:
         #
-        #   assert_field_default_value :string, "MyString"
-        def assert_field_default_value(attribute_type, value)
+        #   assert_field_example_value :string, "MyString"
+        def assert_field_example_value(attribute_type, value)
```

Comment:
This is public API so we would need to add either an alias or deprecate the old method. Same for the name on the attribute.

Reply:
@rafaelfranca I've added the deprecation warning for both.